.github/renovate.json

```diff
@@ -12,9 +12,7 @@
         {
             "customType": "regex",
             "versioningTemplate": "{{#if versioning}}{{versioning}}{{else}}semver{{/if}}",
-            "fileMatch": [
```

Comment:
we probably need to migrate something in `.kres.yaml`

Reply:
https://github.com/siderolabs/kres/commit/6def37e35a7236833d04713c5b4e92d26c8e2b50